pkg/internal/third_party/go-json-experiment/json/example_orderedobject_test.go

```diff
@@ -9,7 +9,7 @@ import (
 	"log"
 	"reflect"
 
-	"k8s.io/kube-openapi/pkg/internal/third_party/go-json-experiment/json"
+	"github.com/go-json-experiment/json"
```

Comment:
all of these need to be rewritten to our imported package or we'll take a dependency on the external repo (as showed up in the CI go.mod verification failure)